package.json

```diff
@@ -25,23 +25,23 @@
     "url": "git://github.com/cucumber/cucumber-electron.git"
   },
   "dependencies": {
-    "ansi-to-html": "^0.6.14",
+    "ansi-to-html": "^0.6.15",
     "electron-window": "^0.8.1"
   },
   "peerDependencies": {
-    "@cucumber/cucumber": "^7.0.0",
-    "electron": ">=8.2.0 <12"
+    "@cucumber/cucumber": "^7.2.1",
+    "electron": "^13.0.1"
```

Comment:
This contradicts the README, doesn't it? Shouldn't it be 11+?

Reply:
Let's only support the latest version to avoid bugs in older versions.